### PR TITLE
Pol 236 aws ec2 compute plugin

### PR DESCRIPTION
**Pull request by gopinathmahakud1**
### Description

Update the AWS EC2 Plugin to work with the Resource Browser/Plugin Registration.

### Issues Resolved

Updated required changes.

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

image

image

### CODE REVIEW

File: aws/rs_aws_compute/changelog.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 Compute Plugin changelog
+
+v1.6
```
Comment (cdwilhelm) [inline aws/rs_aws_compute/changelog.md]: version should be 2.0

Comment (gopinathmahakud1) [reply aws/rs_aws_compute/changelog.md]: I have addressed.

File: aws/rs_aws_compute/aws_compute_plugin.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
-name 'aws_compute_plugin'
+name 'Amazon EC2 Service (Amazon EC2)'
 type 'plugin'
 rs_ca_ver 20161221
-short_description "Amazon Web Services - EC2 Plugin"
+short_description "Amazon Web Services (AWS) - Amazon EC2"
 long_description "Version 1.5"
```
[the remark this answers is not in the document]
Comment (gopinathmahakud1) [reply aws/rs_aws_compute/aws_compute_plugin.rb]: I have addressed.

**Review by cdwilhelm — DISMISSED** [review]
The outputs need to be standardized with id, name, region, status, tags.  You'll need to transform the outputs from instanceId to id.  some resources do not have a name, so you should use the name tag for the name output.  let me know if you need clarification on this.

**Comment by cdwilhelm** [comment]
@gopinathmahakud1   there are some conflicts with this file.

**Review by rshade — APPROVED** [review]
LGTM

**Pull request merged**